recipe/meta.yaml

```diff
@@ -101,9 +104,9 @@ outputs:
     requirements:
       run:
         - {{ pin_subpackage('dlib', exact=True) }}
-    test:
-      imports:
-        - dlib
+    test:  # [not (aarch64 and cuda_compiler_version != "None")]
+      imports:  # [not (aarch64 and cuda_compiler_version != "None")]
+        - dlib  # [not (aarch64 and cuda_compiler_version != "None")]
```

Comment:
Just so I understand, is this just because the `import` doesn't work when cross-compiling or does the `import` call `cuInit` (so requires the GPU and driver library)

Reply:
If I recall correctly, aarch64 fails because of the GLIBC 2.28 issue, which only appears at test time.

Reply:
Ah ok. Would it make sense to add a comment?